frontend/views/user_defined/_show.html.erb

```diff
@@ -1,6 +1,8 @@
 <%
   section_id = "user_defined" if section_id.blank?
- 
```

Comment:
For some reason the digital object read-only view didn't have `context` defined, so it was throwing an exception.  Overriding it here.